src/pybind/mgr/cephadm/services/jaeger.py

```diff
@@ -0,0 +1,81 @@
+from typing import List, Any, Tuple, Dict, cast
+from cephadm.services.cephadmservice import CephadmService, CephadmDaemonDeploySpec
+from ceph.deployment.service_spec import TracingSpec
+from mgr_util import build_url
+
+class ElasticSearchService(CephadmService):
+    TYPE = 'elasticsearch'
+    DEFAULT_SERVICE_PORT = 9200
+
+    def prepare_create(self, daemon_spec: CephadmDaemonDeploySpec) -> CephadmDaemonDeploySpec:
+        assert self.TYPE == daemon_spec.daemon_type
+        daemon_spec.final_config, daemon_spec.deps = self.generate_config(daemon_spec)
+        return daemon_spec
+
+    def generate_config(self, daemon_spec: CephadmDaemonDeploySpec) -> Tuple[Dict[str, Any], List[str]]:
+        assert self.TYPE == daemon_spec.daemon_type
+        return {}, []
```

Comment:
How can we avoid this boilerplate?

Reply:
done [c8eb218](https://github.com/ceph/ceph/pull/44493/commits/c8eb218efff5f003a3b7842d4b93adc980a1c042)